BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.h

```diff
@@ -10,6 +10,11 @@
 @property (nonatomic, readwrite, assign) BOOL requestAllFileFields;
 @property (nonatomic, readwrite, strong) NSArray *notMatchingEtags;//If-None-Match: Array of strings representing etag values
 
+/*
+ * Set this string to "[hls]" request a representation endpoint for HLS video streaming.
```

Comment:
Per our offline conversation. Moving the request construction into our content SDK file request. Until we integrate representations into our app, we will make this a high definition video request. A bool will serve this need.